Tuto-GUDHI-kPDTM-kPLM.ipynb

```diff
@@ -25,7 +25,6 @@
     "In this page, we present three methods to construct approximations of $d_{\\mathcal{K}}$ from a possibly noisy sample $\\mathbb{X}$. The first approximation is the well-known distance-to-measure (DTM) function of [**Chazal11**]. The two last methods are new. They are based on the following approximations which sublevel sets are unions of $k$ balls or ellispoids: the $k$-PDTM [**Brecheteau19a**] and the $k$-PLM [**Brecheteau19b**].\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    dtm = DistanceToMeasure(q, implementation = 'ckdtree', metric='euclidean')
</pre><p>I generally prefer to avoid specifying redundant stuff. <code>DistanceToMeasure(q)</code> is sufficient and avoids overloading the reader with noise.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GUDHI/TDA-tutorial/pull/50/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>